web_src/js/features/repo-legacy.js

```diff
@@ -355,23 +355,28 @@ async function onEditContent(event) {
           const input = $(`<input id="${data.uuid}" name="files" type="hidden">`).val(data.uuid);
           $dropzone.find('.files').append(input);
         });
-        this.on('removedfile', (file) => {
+        this.on('removedfile', async (file) => {
           if (disableRemovedfileEvent) return;
           $(`#${file.uuid}`).remove();
           if ($dropzone.attr('data-remove-url') && !fileUuidDict[file.uuid].submitted) {
-            $.post($dropzone.attr('data-remove-url'), {
-              file: file.uuid,
-              _csrf: csrfToken,
-            });
+            try {
+              const params = new URLSearchParams();
+              params.append('file', file.uuid);
+              await POST($dropzone.attr('data-remove-url'), {data: params});
```

Comment:
```suggestion
              await POST($dropzone.attr('data-remove-url'), {data: new URLSearchParams({file: file.uuid})});
```

I think this is the third time I make this suggestion now. When will you learn? 😆

Reply:
Oof I promise it's the last time :)